README.md

```diff
@@ -175,9 +175,9 @@ the [`JWT` authentication type](https://trino.io/docs/current/security/jwt.html)
 
     ```python
     from sqlalchemy import create_engine
-    
+
     engine = create_engine("trino://<username>@<host>:<port>/<catalog>/<schema>?access_token=<jwt_token>")
-  
+
```

Comment:
extract no-op changes to separate commit.

Reply:
@hashhar this is likely a default from my editor. I'll revert it. Adding the `prettier` pre-commit hook helps to ensure files are formatted in a consistent manner which long term would prevent these types of no-op changes.